### DISCUSSION TITLE
Can't connect to Goerli/Mumbai

### DISCUSSION THREAD

**Discussion by DenysRakhuba** [phantom/docs#99] [Help & Support] [answered]
### Summary

Hello,
I'm currently trying to add Phantom Wallet support on EVM chains and having problems with it on Testnet. I'm able to get account address, but can't get a balance on any of testnet chains (Goerli, Mumbai). I've tried to toggle Testnet mode inside wallet itself, but it seems that I'm getting connected to Ethereum network all the time, because that's what I'm getting as a return from ethereum.networkVersion. 
I've also tried to trigger chain changing as it implemented in Metamask, but it seems it's not supported on your side. 
```
await this.ethereum.request({
  method: 'wallet_switchEthereumChain',
  params: [{chainId: this.chainParams.chainId}]
});
```

Any suggestions? Other wallets on EVM works as supposed. Would be glad to get any response. Thanks

### Steps to Reproduce

After connecting I'm getting next logs:
```
dapp selected chainId: 0x5
ethereum.networkVersion: 0x1
balance from web3.eth.getBalance(account): 0
```

and this error
`VM16820:1 ERROR Error: Returned values aren't valid, did it run Out of Gas? You might also see this error if you are not using the correct ABI for the contract you are retrieving data from, requesting data from a block number that does not exist, or querying a node which is not fully synced.`

And I'm sure I'm at testnet and have propers tokens. 
<img width="359" alt="Screen Shot 2023-09-13 at 4 27 15 PM" src="https://github.com/phantom/docs/assets/16452697/8f67194a-52ab-43c0-a371-a91f2ce6bce5">

### Phantom Version

23.12.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hey @DenysRakhuba,

Thank you for writing in!

I was able to reproduce this bug. We are looking into fixing the issue, I will keep you updated with the status.

**Comment by DenysRakhuba** [comment]
@adamdelphantom Hello!
Any updates on this?
Thanks

**Reply by adamdelphantom** [reply]
Hey @DenysRakhuba, no meaningful updates yet. We intend to address the issue but I don't have an eta for you yet. As soon as I know I will update you here.

**Comment by MD-Dhaval-Saxena** [comment]
Any update on this??

**Reply by adamdelphantom** [reply, marked as answer]
@MD-Dhaval-Saxena @DenysRakhuba The fix for this should be out in approx 3 weeks. It's targeted for 23.16.0 which is the release after next.

Please keep an eye on our changelings in [announcements.](https://github.com/orgs/phantom/discussions/categories/announcements)
